boardgame-ui/src/pages/GuestHomePage.vue

```diff
@@ -15,16 +15,123 @@
         </div>
       </section>
 
-      Lorem ipsum..
+      <div class="container">
+        <section class="content app-description">
+          <h2>Connecting board game players</h2> 
+          <p><i18n path="app.appName"/> is an application for those who like playing to board games. It offers several features for connecting players and enhancing the board game playing experience.</p>
```

Comment:
* "playing board games" 😁

Reply:
Oops, actually had it right the first time..